### DISCUSSION TITLE
TS: tsconfig.json support, include support

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [rome/tools#4655] [Q&A]
I use a monorepo where I generate types from a graphql schema und graphql operations. It is outsourced into a lib that is included in all tsconfig.json files so that works.
rome does not offer such an include nor is reading the tsconfig. is there any way to provide rome with additional type informations or files that hold them at this point?

**Comment by ematipico** [comment]
We don't have these functionalities yet 

The tsconfig support won't be happening anytime soon.

The include file probably yes. Still, I didn't understand your use case
